packages/fe-container/src/components/top-bar/TopBar.test.tsx

```diff
@@ -94,7 +96,7 @@ describe('TopBar tests', function () {
     )
     const image = screen.getAllByTestId('company-logo')[0]
     await userEvent.click(screen.getByTestId('dark-theme-toggle'))
-    expect(image).toHaveAttribute('src', 'https://raw.githubusercontent.com/lauragift21/giftegwuenu.dev/master/src/assets/img/logo-light.png')
+    expect(image).toHaveAttribute('src', theming.logo.url_dark_image)
```

Comment:
I'd set the `url_dark_image` in a costant so that, if someone removed it from the props, that person would be aware of this test. 
Otherwise, what would happen is that both `theming.logo.url_dark_image` and `image` would be undefined and this test would pass, but it wouldn't test what it's supposed to.

Reply:
Since url_darl_image is not mandatory, do you suggest splitting the test one for when it is passed and the other when not?